### ISSUE TITLE
Switch from Paperclip to ActiveStorage

### ISSUE THREAD

**Issue by CyanoKobalamyne** [TheMITTech/thetech.com#558]
Paperclip was deprecated in May 2018: https://thoughtbot.com/blog/closing-the-trombone. Rails ActiveStorage can be used instead, which also has an AWS S3 backend.

Example article that describes a large-scale migration: https://medium.com/sortlist-engineering/migrating-from-paperclip-to-activestorage-a-different-approach-4bffa2260e17.

**Comment by bkettle** [comment]
May be less relevant in light of #575. Seems like it would still eventually be good to move to ActiveStorage, but the immediate need is avoided for now.

**Comment by CyanoKobalamyne** [comment]
Yes, I think the benefit is a reduced dependency tree size (and hence reduced exposure to bugs). But there are likely many higher-priority issues...
